cognite/extractorutils/unstable/core/base.py

```diff
@@ -22,6 +22,11 @@ class MyConfig(ExtractorConfig):
         another_parameter: int
         schedule: ScheduleConfig
 
+    class MyMetrics(BaseMetrics):
+        def __init__(self, extractor_name: str, extractor_version: str):
+            super().__init__(extractor_name, extractor_version)
+            self.custom_counter = Counter("custom_counter", "A custom counter")
```

Comment:
This looks like it was added to show things work, but surely you don't want this in the production build?